src/event.rs

```diff
@@ -454,12 +398,11 @@ pub enum MouseButton {
 impl MouseButton {
     #[cfg_attr(any(feature = "test", test), allow(dead_code))]
     fn from_button(button: glutin_event::MouseButton) -> Option<Self> {
-        use glutin_event::MouseButton::*;
         match button {
```

Comment:
If you add the `#[deny]` attribute I mentioned above, we can avoid prefixing these lines too.